src/vm/dwbucketmanager.hpp

```diff
@@ -1062,37 +1062,45 @@ void BaseBucketParamsManager::FindFaultingMethodInfo()
         if (throwable != NULL)
         {
             StackTraceArray traceData;
-            EXCEPTIONREF(throwable)->GetStackTrace(traceData);
-
+            ZeroMemory(&traceData, sizeof(traceData));
             GCPROTECT_BEGIN(traceData);
 
-            size_t numElements = traceData.Size();
-
-            ContractFailureKind kind = GetContractFailureKind(throwable);
-
-            // skip frame 0 since we already know it's part of code contracts
-            for (size_t index = 1; index < numElements; ++index)
+            EX_TRY
```

Comment:
Whitespace changes here due to addition of `EX_TRY` to make sure exceptions didn't escape in a `NOTHROW` method

Reply:
Exception would be from obtaining a lock so probably in trouble if this state is reached anyway